ruby-challenges.rb

```diff
@@ -20,6 +39,22 @@
 # Expected output: 'Sgt Pppr'
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
+ 
+
+# PsuedoCode
+# def variable conects to string
+# use a string split 
+# use a string delete
+# use a string join
+# Push p
+def remove_vowels(string)
+  string = string.split("")
+  string.delete("aeiouAEIOU")
+  string = string.join(" ")
+end
+
+p remove_vowels(album1, album2, album3)
```

Comment:
If you have one param, you will only pass one argument when you invoke the method.